### PR TITLE
fix: Enable dualstack/v6 in probes

### PR DESCRIPTION
**Pull request by p-fruck**
Please ensure your pull request adheres to the following guidelines:
- [ ] explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Document your Changes in the `CHANGELOG.md` file of the respected chart as well as the `Chart.yaml`

Continuation of https://github.com/SonarSource/helm-chart-sonarqube/pull/404, fixing dualstack/ipv6 probes for SonarQube CE variant only

### CODE REVIEW

File: charts/sonarqube/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ All changes to this chart will be documented in this file.
 * Run the initSysctl init-container as root to prevent 'permission denied' issues
 * Add revisionHistoryLimit configuration for SonarQube application Deployment ReplicaSets & StatefulSets
 * Update the security contexts to use root as group ID
+* Add support for IPv6 single stack clusters
```
Comment (jCOTINEAU) [inline charts/sonarqube/CHANGELOG.md:10]: This line can be removed ?

**Review by jCOTINEAU — CHANGES_REQUESTED** [review]
One minor comment.

Huge thanks for taking the time to contribute to the SonarQube helm chart!

File: charts/sonarqube/CHANGELOG.md (hunk 1 above)
Comment (p-fruck) [reply charts/sonarqube/CHANGELOG.md]: good catch, I updated the changelog accordingly

**Review by jCOTINEAU — APPROVED** [review]
lgtm !

**Pull request merged**